src/applications/personalization/profile/tests/components/paperless-delivery/CheckboxAlert.unit.spec.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { cleanup, render } from '@testing-library/react';
+import { expect } from 'chai';
+import { CheckboxAlert } from '../../../components/paperless-delivery/CheckboxAlert';
+
+describe('CheckboxAlert', () => {
+  afterEach(() => {
+    cleanup();
+  });
+
+  it('should not render', () => {
```

Comment:
I think in this tests would be more readable with some context about the conditions, like `should not render when neither success nor error is provided`. I think this becomes more of an issue with some of the tests further down that have more involved setup.

Reply:
Agreed